<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>781</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="storepage">
     <widget class="QGroupBox" name="BookOfMoney">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>221</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Book of Money</string>
      </property>
      <widget class="QLabel" name="boofMoney">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>81</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QLabel" name="bookofmoneyPrice">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>$ 21</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="PsychologyOfMoney">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>231</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Psychology of Money</string>
      </property>
      <widget class="QLabel" name="psyofMoney">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>81</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QLabel" name="psyofmoneyPrice">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>$ 29</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="RichDadPoorDad">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>221</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Rich Dad Poor Dad</string>
      </property>
      <widget class="QLabel" name="richDad">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>81</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QLabel" name="richdadPrice">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>$ 25</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="viewcart">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>300</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>View Cart</string>
      </property>
     </widget>
     <widget class="QLabel" name="shopName">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Money store</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="SpeakAboutMoney">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>330</y>
        <width>231</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Speak about Money</string>
      </property>
      <widget class="QLabel" name="speakAboutMoney">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>81</width>
         <height>131</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QLabel" name="speakaboutmoneyPrice">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>$ 12</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="viewcart_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Money Store</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Items in cart</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>421</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="payment">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Money Store</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Payment</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="thankyou">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Money Store</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Order Placed</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
